packages/core/src/external/athenahealth/index.ts

```diff
@@ -181,6 +185,57 @@ class AthenaHealthApi {
     }
   }
 
+  async getPatientViaSearch({
+    cxId,
+    patientId,
+  }: {
+    cxId: string;
+    patientId: string;
+  }): Promise<PatientResource | undefined> {
+    const { log, debug } = out(`AthenaHealth search - AH patientId ${patientId}`);
+    try {
+      const data = {
+        _id: this.createPatientId(patientId),
+        "ah-practice": this.createPracticetId(this.practiceId),
+      };
+      const patienSearchtUrl = "/Patient/_search";
```

Comment:
nit: place this outside the function

Reply:
I'll leave it as is -- it's not a "root" URL that would be defined more globally IMO.

Reply:
ya, but still no point in reinstantiating this on every function call